### ISSUE TITLE
Transition.Out and ImageBackground / Images

### ISSUE THREAD

**Issue by LiVe** [software-mansion/react-native-reanimated#580]
Hey everyone,

I'm currently using react-navigation-animated-switch which uses this package for transitions.
I've defined the following transition:
```
<Transition.Together>
   <Transition.Out
      type="fade"
      durationMs={6000}
      interpolation="easeIn"
   />
   <Transition.In
      type="fade"
      durationMs={6000}
      interpolation="easeIn"
   />
</Transition.Together>
```

Now the transition itself works quite fine, except for one usecase. If i'm using a ImageBackground component (as root) on my "leaving" screen with Images within the layout, the images (background as well as other images) disappear right away. Other child components are fine and background colors seam to stay too.

I hope somebody could point me in the right direction. Thanks!

**Comment by ninjz** [comment]
I think I may have the same issue. Images that are a child of the Transitioning View is not being animated.

**Comment by ninjz** [comment]
I think my issue with the Image not animating was due to my images being too large. I was able to get them to animate on transition by preloading the images. I guess when I previously commented, the Image was still loading and in that case, only appeared after the transition so it seemed like the animation never occurred.

**Comment by LiVe** [comment]
i don't think this has anything todo with loading times on my end. The image is visible on the current view and once it transitions out is immediately gone. even if i increase the duration to something very long, it won't ever show.

**Comment by LiVe** [comment]
https://github.com/react-navigation/react-navigation/issues/7241

**Comment by ninjz** [comment]
@LiVe I too had images that also behaved that way and initially thought it had nothing to do with the loading times. One image was slow to render and it was what led me onto trying to prefetch the image. So there are two things I did: 

1. Making sure the parent TransitioningView has a defined height.
2. Prefetching the images using the method shown here https://github.com/DylanVann/react-native-fast-image/issues/395#issuecomment-453850446

I noticed I had some weird behavior when the parent TransitioningView did not have a defined height. I'd suggest giving this a try and seeing what happens.

**Comment by jakub-gonet** [comment]
We discussed Transitions internally in our team. Considering that v2 makes creating animations easier and Transitions was an experimental API for a long time, we decided to leave Transitions as they are.
 
We won’t actively fix bugs in them, but we’ll happily merge any PR with fixes. We’re also thinking about rewriting Transitions in v2 sometime in the future.

I’m closing this issue, but if you have further questions, I will be happy to help.
